__tests__/html2/livestream/activityOrder.html

```diff
@@ -37,15 +37,15 @@
           return false;
         };
 
-        const App = () => (
-          <Composer directLine={directLine} store={store}>
-            <BasicWebChat />
-            <Monitor />
-          </Composer>
+        render(
+          createElement(
```

Comment:
I'm ok with `createElement` but it's also common to use `h`

Reply:
Let's keep it for now. I am wondering if we can go JSX Runtime with 16.8.6 by rebuilding `_jsx` function.

Thinking about using esm.sh/run with `<script type="text/babel">`.